active_learning_multiannotator/active_learning_single_annotator.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
do not mention the word annotator in this notebook, except in one place somewhere near the top:

> This notebook considers settings where you want to collect at most one label for each example (i.e. the standard active learning setting). In constrast, the [Active Learning with Multiple Data Annotators](https://github.com/cleanlab/examples/blob/master/active_learning_multiannotator/active_learning.ipynb) demonstrates how to do active learning in settings where you have multiple data annotators who can re-label data previously labeled by other annotators (i.e. to check their labels if we are worried they are noisy).

Reply:
there are many places I see the word "annotator" in this notebook I find concerning that you just copy/pasted from the other notebook. Need to be sure everything in this notebook is 100% specific to the setting where there's only at most 1 label per datapoint. 

Just to highlight a few cases of this:

> (Active CROWDLAB appropriately estimates the value of collecting new annotations for unlabeled data vs already-labeled data)

- your notebook still mentions `pred_probs_labeled` which seems like an irrelevant concept in this single-label setting.

> we will get the active learning scores for each datapoint (both labeled and unlabeled) by using a combination of the annotators' agremeent and model confidence.